### PR TITLE
DOC: updated files for SS06 errors

### PR DESCRIPTION
**Pull request by willpeppo**

### CODE REVIEW

File: pandas/core/indexes/multi.py [hunk 1]
```diff
@@ -1541,8 +1540,7 @@ def _get_level_values(self, level, unique=False):
 
     def get_level_values(self, level):
         """
-        Return vector of label values for requested level,
```
Comment (WillAyd) [inline pandas/core/indexes/multi.py:1544]: I think should keep the extra information here and in the other docstrings rather than just removing.

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Comment by WillAyd** [comment]
@willpeppo can you fix merge conflict and address comments?

**Comment by WillAyd** [comment]
@willpeppo still active?

**Comment by willpeppo** [comment]
Yes sorry.  Will get back going on documentation shortly.   I’d like to be an ongoing contributor.  Sorry!

**Comment by simonjayhawkins** [comment]
@willpeppo Thanks for the PR. closing as stale for now. LMK when you are ready to continue and i'll reopen.

**Pull request closed without merging**